src/components/CorrectAddresses/styles.js

```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+export const AddressItem = styled.p`
+  margin: 0;
```

Comment:
o, to jest dobry przyklad gdzie bym nie robił styled componentu, tylko dał bezposrednio w komponencie css={{margin: 0}} (css ze styled-components)

Reply:
racja zgadzam się w 100%, lecz dodałem teraz media wiec teraz już zostawię.
Done.